src/sell_ticket.lua

```diff
@@ -1,14 +1,12 @@
--- TODO
--- Edelstein
-
 if target.field == 200000100 then
     selection = self.ask_menu("Hello, I'm the information guide for Orbis Station. Which of our destinations would you like to know more about?", {
         [0] = "Victoria Island",
         [1] = "Ludibrium",
         [2] = "Leafre",
         [3] = "Mu Lung",
         [4] = "Ariant",
-        [5] = "Ereve"
+        [5] = "Ereve",
+		[6] = "Edelstein"
```

Comment:
Indentation in this file and in station_in.lua is a little off

Reply:
i dunno why this is so, looks fine in my IDE